java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/FacetNotExistException.java

```diff
@@ -7,10 +7,20 @@
  * not find a servant for the identity + facet carried by the request.
  */
 public final class FacetNotExistException extends RequestFailedException {
+    /**
+     * Constructs a {@code FacetNotExistException} with the default message.
```

Comment:
What does this mean "with the default message"?

For all the others we just say "Constructs a {@code FacetNotExistException}" without any mention of a default message.